packages/app-project/src/screens/ClassifyPage/components/YourProjectStats/helpers/incrementStats.js

```diff
@@ -0,0 +1,19 @@
+export default function incrementStats(mutate, projectID, userID) {
```

Comment:
`mutate` is specific to `useYourProjectStats()` hook. No API calls are made here. The `data` returned by the custom hook is just updated per classification submitted.